helm-service/controller/action_handler.go

```diff
@@ -42,6 +43,10 @@ func (h *ActionTriggeredHandler) HandleEvent(ce cloudevents.Event) {
 		return
 	}
 
+	// retrieve commitId from sequence
+	extensions := ce.Context.GetExtensions()
+	commitID, _ := types.ToString(extensions["gitcommitid"])
```

Comment:
Should we react to potential errors returned by `types.ToString()`? If there is no realistic scenario where this occurs it's fine to neglect it, but let's add a short comment :)

Reply:
git commit is always stored as a string since we either use cli or get it from our resource service, since this is already a cloudevent type no validation is performed and the function cannot fail. I'll add this as a comment!